### PR TITLE
Show snackbars for information messages

### PR DESCRIPTION
**Pull request by Martynas-S**
@erteriuke touched on a fine point when she said that the user should be informed on successful operations such as for example, transaction creation.

I propose we use snackbars for that because they are very easy to use and look good.

Peek 2019-05-01 13-31

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -54,13 +55,15 @@ const HomeRoutes = () => {
...
 const App = () => {
   return (
-    <Router>
-      <Switch>
-        <GuestRoute path={LoginLocation} Component={Login} />
-        <GuestRoute path={RegisterLocation} Component={Register} />
-        <UserRoute path="/" Component={HomeRoutes} />
-      </Switch>
-    </Router>
+    <SnackbarProvider maxSnack={3}>
```
Comment (Andrius-B) [inline frontend/src/App.tsx]: kodėl čia` {3}`? 🤔

File: frontend/src/Views/Checkout/Checkout.tsx [hunk 2]
```diff
@@ -70,10 +73,18 @@ const Checkout = (props: IProps) => {
     }
     postTransaction(currentTransaction)
       .catch(error => {
-        console.log(error);
+        enqueueSnackbar(
+          "An unspecified error occured when creating the transaction",
+          {
+            variant: "error"
+          }
+        );
```
Comment (Andrius-B) [inline frontend/src/Views/Checkout/Checkout.tsx:72]: Kodėl šitam snackbar tiek daug vertical space kode davei? Dvigubai daugiau nei tam po apačia \\/ 🤣 

_Nes erroras?_

**Review by Andrius-B — APPROVED** [review]

File: frontend/src/Views/Checkout/Checkout.tsx (hunk 2 above)
Comment (Martynas-S) [reply frontend/src/Views/Checkout/Checkout.tsx:72]: Cia `prettier` suformatavo, nes negali eilute virsyti 80 simboliu ;D

File: frontend/src/App.tsx (hunk 1 above)
Comment (Martynas-S) [reply frontend/src/App.tsx]: Va siaip sau, i ka nori kad pakeisciau? :D

File: frontend/src/Views/Checkout/Checkout.tsx (hunk 2 above)
Comment (Andrius-B) [reply frontend/src/Views/Checkout/Checkout.tsx:72]: okay

File: frontend/src/App.tsx (hunk 1 above)
Comment (Andrius-B) [reply frontend/src/App.tsx]: `1` gal? idk, tų snackbar spam tai nėra gerai

**Pull request merged**